dkg/dkgpb/v1/bcast.proto

```diff
@@ -8,7 +8,7 @@ option go_package = "github.com/obolnetwork/charon/dkg/dkgpb/v1";
 
 message BCastSigRequest {
   string id   = 1;
-  bytes  hash = 2;
```

Comment:
Can't we just mark it as `[deprecated = true]` to avoid reusing the field in a wrong way in the future?
Or, we can make it a reserved field: https://protobuf.dev/programming-guides/proto3/#fieldreserved

Reply:
Good call! I'll reserve 2.